src/inspector/main_thread_interface.cc

```diff
@@ -194,8 +194,11 @@ class CrossThreadInspectorSession : public InspectorSession {
 
 class ThreadSafeDelegate : public InspectorSessionDelegate {
  public:
-  ThreadSafeDelegate(std::shared_ptr<MainThreadHandle> thread, int object_id)
-                     : thread_(thread), delegate_(thread, object_id) {}
+  ThreadSafeDelegate(std::shared_ptr<MainThreadHandle> thread,
+                     int object_id,
+                     int is_trusted)
```

Comment:
```suggestion
                     bool is_trusted)
```

(also, the following lines should probably be de-indented to only 4 blanks in front of the `:`, and one line per item)